.github/workflows/pr-test.yml

```diff
@@ -66,7 +66,7 @@ jobs:
      
       - name: Publish Unit Test Results
         uses: docker://ghcr.io/enricomi/publish-unit-test-result-action:v1.6
-        if: always()
+        if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
Will this still run regardless of the failure of a previous step (i.e. if the tests fail)? That's the behavior we want, I'm just not sure if github treats `always()` as special - I guess we'll see

Reply:
@JBAhire we can test this by introducing failure in the test and creating dummy PR.